src/main/java/yoda/Ui.java

```diff
@@ -0,0 +1,78 @@
+package yoda;
+
+/**
+ * Encapsulates the user interface for the chatbot.
+ * @author Tonishka Singh
+ */
+public class Ui {
+    public Ui() {}
+
+    /**
+     * Prints the welcome message when user launches the application.
+     */
+    public void welcome() {
```

Comment:
Maybe call this method printWelcome or displayWelcome or something along those lines to phrase it as a verb?